service-front/module/Application/src/Adapter/DynamoDbKeyValueStore.php

```diff
@@ -121,7 +121,7 @@ public function getItem($key, & $success = null, & $casToken = null)
 
             $success = true;
 
-            return $result['Item']['value']['B'];
+            return $result['Item']['value']['B'] ?? null;
```

Comment:
Not quite sure what the point of this is? You're checking to see if it's null then returning null anyway?

Reply:
If the result does not contain an Item, or the Item does not contain a value, or the value does not contain B, then with PHP 8, we need to ensure that rather than blow up, it returns null.  See https://stitcher.io/blog/php-8-nullsafe-operator

Reply:
I was going to suggest using `isset()` with the ternary operator instead. But have just realized that it would make it longer unnecessarily, when this does the exact same thing.